package/kedro_viz/launchers/utils.py

```diff
@@ -18,7 +18,8 @@ class WaitForException(Exception):
 def _wait_for(
     func: Callable,
     expected_result: Any = True,
-    timeout: int = 60,
+    # [TODO] This is a temporary fix for https://github.com/kedro-org/kedro-viz/issues/1768.
```

Comment:
Do we have an open issue about the long term fix? Whether that's reworking the `KedroSession`, reworking the hooks, or just exploring the solution space because we don't know better for now.

Reply:
Hi @astrojuanlu , The long term fix would involve making Kedro Viz faster. 
https://github.com/kedro-org/kedro-viz/issues/1783, https://github.com/kedro-org/kedro-viz/issues/1808 will help us move towards that and https://github.com/kedro-org/kedro-viz/issues/1742 would be something which we want to explore and build.

Thank you